### PR TITLE
Return null when getLoader() on non-existent identifier

### PR DESCRIPTION
**Pull request by melicerte**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -185,8 +185,16 @@ export class AppComponent implements AfterViewInit {
 
 ## Contribute
 Firstly fork this repo, then clone it and go inside the root of the freshly forked project.
+
+### Install
+`yarn`
+
+then
+
+`yarn add --dev node-sass`
```
Comment (mcsky) [inline README.md]: I don't understand why this line is in the readme. If `node-sass` is needed why don't add it to dependencies?

Comment (melicerte) [reply README.md]: True. @maximelafarie can you confirm we can add node-sass package in dev dependencies without any problem ?

Comment (melicerte) [reply README.md]: Removed until response, this is not related to the current PR

Comment (maximelafarie) [reply README.md]: What was this line? Was it added in the @melicerte's commit? Normally, nothing special has to be added in the `package.json` of the library. 🤔

Comment (melicerte) [reply README.md]: I removed it from commit.

But node-sass must be installed indeed to make demo work. To address in another issue I guess.

**Comment by maximelafarie** [comment]
Done in #12! 😉

**Pull request closed without merging**